### PR TITLE
Update for Tube Cup 2021

### PR DESCRIPTION
**Pull request by Gum-Joe**
Update the site for Tube Cup 2021:

- Added new tube colours
- Made Game a general component, so we can just specify the right link to get data from
- Made descriptions for each round a prop to GameBoard

### CODE REVIEW

**Comment by Gum-Joe** [comment]
Pics:
image
image
image

File: .github/workflows/smoketest.yml [hunk 1]
```diff
@@ -38,11 +38,11 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install dependencies on cache miss
-        run: yarn install --frozen-lockfile
+        run: yarn install
```
Comment (davwheat) [inline .github/workflows/smoketest.yml:41]: Immutable is the replacement for frozen lockfile.

```suggestion
        run: yarn install --immutable
```

File: .github/workflows/smoketest.yml [hunk 2]
```diff
@@ -38,11 +38,11 @@ jobs:
...
         if: steps.yarn-cache.outputs.cache-hit != 'true'
 
       - name: Install project dependencies from cache
-        run: yarn --prefer-offline --frozen-lockfile
+        run: yarn --cached
```
Comment (davwheat) [inline .github/workflows/smoketest.yml]:
```suggestion
        run: yarn --cached --immutable
```

File: .github/workflows/smoketest.yml (hunk 1 above)
Comment (Gum-Joe) [reply .github/workflows/smoketest.yml:41]: Yeah I did that. The build had a checksum error I couldn't solve.

Comment (Gum-Joe) [reply .github/workflows/smoketest.yml:41]: See 25b28fbea5e519f1e7256da80805310a11ccfd5c

Comment (davwheat) [reply .github/workflows/smoketest.yml:41]: Likely needs a lockfile generated on Linux.

Comment (Gum-Joe) [reply .github/workflows/smoketest.yml:41]: VSCodespaces to the rescue then!

**Review by davwheat — APPROVED** [review]

**Pull request merged**